app/components/avatar/avatar.rb

```diff
@@ -1,15 +1,33 @@
 # frozen_string_literal: true
 
 module Avatar
-  class Avatar < ViewComponent::Base
-    include ComponentHelper
+  class Avatar < ApplicationComponent
+    FALLBACK_BASE_URL = '/avatars'
+    FALLBACK_IMAGES = [
+      'fallback-cat.jpg',
+      'fallback-dog.jpg',
+      'fallback-otter.jpg',
+      'fallback-seal.jpg',
+      'fallback-squirrel.jpg',
+    ]
 
-    def initialize(url:)
+    def initialize(url: nil, key: 0)
```

Comment:
```suggestion
    def initialize(url: nil, animal: nil)
```

Reply:
Why would you want to specify a concrete fallback image?